cmd/controller/main.go

```diff
@@ -78,13 +77,16 @@ func main() {
 	if err := images.Validate(); err != nil {
 		log.Fatal(err)
 	}
-
-	controller.DefaultThreadsPerController = *threadsPerController
-
-	cfg := sharedmain.ParseAndGetConfigOrDie()
+	if cfg.QPS == 0 {
+		cfg.QPS = 2 * rest.DefaultQPS
+	}
+	if cfg.Burst == 0 {
+		cfg.Burst = rest.DefaultBurst
+	}
+	// FIXME(vdemeester): this is here to not break current behavior
 	// multiply by 2, no of controllers being created
-	cfg.QPS = 2 * float32(*qps)
-	cfg.Burst = 2 * *burst
+	cfg.QPS = 2 * cfg.QPS
+	cfg.Burst = 2 * cfg.Burst
```

Comment:
@tektoncd/core-maintainers This is a bit weird but it's there to keep the same behavior as of today. Because the flag default value is defined in knative/pkg, we don't have the control over it, so we detect if the default value are passed or not. The `2 * …` I am not sure about why we are doing this, but I didn't want to change the behavior here.